config/v1/types_cluster_operator.go

```diff
@@ -173,6 +173,9 @@ const (
 	// of what the administrator should do to allow the operator to successfully update.  A missing condition, True,
 	// and Unknown are all treated by the CVO as allowing an upgrade.
 	OperatorUpgradeable ClusterStatusConditionType = "Upgradeable"
+
+	// Disabled indicates that the operator is disabled.
```

Comment:
I would add something like "explicitly configured to remove its operators" or something to explain what `Disabled` means, and ideally add some guidance about the sort of messages should be set in that case.

Reply:
I propose `Disabled indicates that the operator is explicitly configured to be disabled or is not meant to run on this platform.`

Insight operator can be disabled with a ConfigMap, this can be changed by the user.
For baremetal operator, it is more by-design. https://github.com/openshift/cluster-baremetal-operator/blob/a7f05f27579c222a10b63f63e77fa66b3729345e/controllers/provisioning_controller.go#L97

CCO falls in the same category as baremetal operator.

Reply:
your text sounds good to me.  What about suggested reasons and messages?  I'd prefer a recommendation of a single sentence of ten words or less or some such, so folks building UXes around this message have some idea of what to expect (although they should still guard against operators potentially setting huge, multi-line messages as corner cases).